test/testdata/generate.sh

```diff
@@ -58,58 +48,55 @@ scaffold_test_project() {
     header_text 'Creating APIs ...'
     $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
     $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false --force
-    $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
+    $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation --make=false
```

Comment:
if is make false then we cannot validate the --force properly since the manifest will not be gen.

Reply:
The `--force` flag doesn't act different if there is or there isn't any manifest. It has to do with the tracked resources and the scaffolded files.